Lib/fontTools/merge.py

```diff
@@ -771,7 +771,10 @@ def merge(self, m, tables):
 	assert len(tables) == len(m.duplicateGlyphsPerFont)
 	for i,(table,dups) in enumerate(zip(tables, m.duplicateGlyphsPerFont)):
 		if not dups: continue
-		assert (table is not None and table is not NotImplemented), "Have duplicates to resolve for font %d but no GSUB: %s" % (i + 1, dups)
+		if table is None or table is NotImplemented:
+			log.warning("Have non-identical duplicates to resolve for font %d but no GSUB. Are duplicates intended?: %s" % (i + 1, dups))
```

Comment:
Minor optimisation: when interpolating strings in logging messages one usually uses the star-args at the end instead of `% (...)`, this way the string is interpolated only if/when the log message is actually emitted.

```diff
- log.warning("Have non-identical duplicates to resolve for font %d but no GSUB. Are duplicates intended?: %s" % (i + 1, dups))
+ log.warning("Have non-identical duplicates to resolve for font %d but no GSUB. Are duplicates intended?: %s", i + 1, dups)
```